<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record X Vs Y</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QCommandLinkButton" name="titleButton">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="startLabel">
     <property name="text">
      <string>Start Value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="startBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="stopLabel">
     <property name="text">
      <string>Stop Value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="stopBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="stepLabel">
     <property name="text">
      <string>Step Size</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="stepBox"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>myframe</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>interval</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="intervalBox">
     <property name="suffix">
      <string> mS</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255))
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>START</string>
     </property>
     <property name="icon">
      <iconset resource="resplus.qrc">
       <normaloff>:/control/play.svg</normaloff>:/control/play.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="stopButton">
     <property name="text">
      <string>STOP</string>
     </property>
     <property name="icon">
      <iconset resource="resplus.qrc">
       <normaloff>:/control/stop.svg</normaloff>:/control/stop.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>CLEAR</string>
     </property>
     <property name="icon">
      <iconset resource="resplus.qrc">
       <normaloff>:/control/reset.svg</normaloff>:/control/reset.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>13</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resplus.qrc"/>
 </resources>
 <connections/>
</ui>
